File: Plugins/Lpp.Dns.General.CriteriaGroupCommon/Models/Terms/AgeStratifier.cs
using System.Runtime.Serialization;

namespace RequestCriteria.Models
{
    [DataContract]
    public class AgeStratifierData : TermData
    {
        [DataMember]
        public AgeStratifierTypes AgeStratifier { get; set; }
    }

    public enum AgeStratifierTypes
    {
        [EnumMember]
        NotSpecified = -1,
        [EnumMember]
        None = 0,
        [EnumMember]
        Ten = 1,
        [EnumMember]
        Seven = 2,
        [EnumMember]
        Four = 3,
        [EnumMember]
        Two = 4
    }
}

File: Plugins/Lpp.Dns.General.Metadata/Content/Models/Terms/AgeStratifier.ts
/// <reference path="../../../../../Lpp.Mvc.Composition/Lpp.Mvc.Boilerplate/jsBootstrap.d.ts" />
/// <reference path="../Terms.ts" />

module RequestCriteriaModels {
    export interface IAgeStratifierTermData extends RequestCriteriaModels.ITermData {
        AgeStratifier: AgeStratifierTypes;
    }

    export enum AgeStratifierTypes {
        NotSpecified = -1,
        None = 0,
        Ten = 1,
        Seven = 2,
        Four = 3,
        Two = 4
    }
}